docs/getting-started/clients/desktop/update.md

```diff
@@ -46,7 +49,60 @@ process without messing around in a GitHub repo.
 4.  Publish the new version using: `npm run publish:win:dev`
 5.  The app should now prompt for an update.
 
-Note: This can also be done on Linux, just use `npm run publish:lin` for steps 1 and 4.
+Note: This can also be done on Mac and Linux, just use `npm run publish:mac` and
+`npm run publish:lin` for steps 1 and 4.
+
+## Staged rollouts
+
+The process for testing staged rollouts is a bit cumbersome, as it requires updating the update
+manifests and uploading them manually. The detailed steps on how to do a staged update are as
+follows:
+
+- Create and publish an update as mentioned in the previous section.
+- Open the MinIO object browser and navigate to the bucket.
+- Download the update manifest. This will be the file named
+  `latest.yml`/`latest-mac.yml`/`latest-linux.yml`, depending on your platform.
+- Open the file in your preferred editor, and add a line at the end in the format
+  `stagingPercentage: X`, as shown in the image below.
+  - `X` is the percentage of users that will get the update. It must be an integer greater than zero
+    and equal or less than 100.
+- Upload the file to the bucket again by drag-and-dropping it into the file list.
+
+  ![](./minio-manifest-rollout.png)
+
+When checking updates using staged rollouts, `electron-updater` will use a random persistent
+per-installation GUID and derive a rollout percentage from it. Then it will refuse to apply any
+updates where the staging percentage in the downloaded update manifest is less than the
+per-installation percentage. This randomness complicates testing, but it's possible to replace the
+random GUID by a fixed one to guarantee reproducible tests:
+
+- Navigate to the application's data folder. In Windows this is
+  `C:\Users\<user>\AppData\Roaming\Bitwarden`
+- Open the file called `.updaterId` in your favorite editor and replace the GUID inside by
+  `c8beb766-67f5-4058-b6a1-7031fef1c31a`.
+- This GUID has a rollout percentage of 99.5, which means that:
+  - Using `stagingPercentage: 100` will guarantee the client is part of the staged rollout.
+  - Using any other value will guarantee the client is NOT part of the rollout.
```

Comment:
Weelll you only have a 99.5% chance of this being true 🤣 

```suggestion
  - Using any other value almost guarantee the client will NOT be part of the rollout.
```

Reply:
Supposedly the value is rounded up to an integer before doing the calculation, but yeah better not be as absolute about it lol